jersea/src/components/index.js

```diff
@@ -1,4 +1,6 @@
 export {ProductCard} from './Products/ProductCard/ProductCard.jsx'
 export {ProductFilter} from './Products/ProductFilter/ProductFilter.jsx'
 export {ProductList} from './Products/ProductList/ProductList.jsx'
-export {Header} from './Header/Header.jsx'
\ No newline at end of file
+export {Header} from './Header/Header.jsx'
```

Comment:
Try using index file for reusable components

Reply:
sure will do it